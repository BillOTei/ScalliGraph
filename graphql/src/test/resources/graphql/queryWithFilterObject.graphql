query filterPerson {
  allPeople {
    filter(
      filter: {
        name_contains: "o"
      }
    ) {
      age {
        sort {
          toList
        }
      }
    }
  }
}