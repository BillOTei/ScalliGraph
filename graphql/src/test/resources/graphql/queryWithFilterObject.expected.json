{
  "data": {
    "allPeople": {
      "filter": {
        "age": {
          "sort": {
            "toList": [
              29,
              32
            ]
          }
        }
      }
    }
  }
}